backend/estimate-service/src/main/java/org/egov/works/service/EnrichmentService.java

```diff
@@ -22,11 +21,13 @@ public class EnrichmentService {
 
     private EstimateServiceUtil estimateServiceUtil;
     private IdGenRepository idGenRepository;
+    private EstimateServiceConfiguration config;
 
     @Autowired
-    public EnrichmentService(EstimateServiceUtil estimateServiceUtil,IdGenRepository idGenRepository) {
+    public EnrichmentService(EstimateServiceUtil estimateServiceUtil, IdGenRepository idGenRepository, EstimateServiceConfiguration config) {
```

Comment:
Why the need to write a constructor? Won't this work without a constructor? 
Just Autowire every member component.

Reply:
Yes ..it'll work but in digit they are using constructor injection everywhere.

Reply:
done.